README.md

```diff
@@ -45,7 +45,7 @@ $ docker run -p 8080:8080 taco
 ### Build for the local OS
 ```shell
 % cd cmd/tacod
-% go get -t
+% dep ensure -vendor-only
```

Comment:
Is the `-vendor-only` flag necessary?

Reply:
Yes, otherwise it’ll recalculate the dependencies rather than use what we have tested locally, which is specified by the lock file.

Reply:
No, this is exactly the opposite of what we've been working through.

Reply:
Justin, have you done #94 yet? I think the rest of the group has - and I don't know if what issues you are having due to your setup are occurring for others. I'd say close this PR until you've gone through #94 and we have some more concrete, shared knowledge on usage of dep.

Reply:
Here is some evidence of why `dep ensure` gives a user a version that is not the same as what we've developed on: https://github.com/sul-dlss-labs/taco/pull/151